MapboxNavigation/RouteStepFormatter.swift

```diff
@@ -0,0 +1,53 @@
+import Foundation
```

Comment:
We have a RouteStepFormatter in NavUI as well, let's get rid of the one in NavUI since this is already public.